css/style.css

```diff
@@ -0,0 +1,1159 @@
+/*Обнуление*/
+*{padding:0;margin:0;border:0;}
```

Comment:
Codacy found an issue: [Expected single space after ";" in a single-line declaration block (declaration-block-semicolon-space-after)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)